manuscript.Rmd

```diff
@@ -73,30 +73,31 @@ should not be longer than 1-1.5 pages, typically 4-5 paragraphs, each
 
 Background
 ----------
-Trauma/Injury is a substantial global health problem that disproportionately affects low and middle-income countries (LMIC), resulting in significant mortality and morbidity @traumaLMIC. Despite having only 1% of the global vehicle fleet, a disproportionate 10% of global traffic casualties occur in India @indiantrauma. The annual mortality from injury in India amount to one million casualties @onemilliondead. Half of trauma mortality in India occur in-hospital, and in-hospital trauma mortality in Indian hospitals was double that of high-income countries @indiantrauma. 
+Trauma/Injury <!-- Define trauma here, as there is a difference between trauma and Injury. Trauma can be defined as physical injury and the bodys physiological response --> is a substantial global health problem that disproportionately affects low and middle-income countries (LMIC), resulting in significant mortality and morbidity @traumaLMIC. Despite having only 1% of the global vehicle fleet, a disproportionate 10% of global traffic casualties occur in India @indiantrauma. The annual mortality from injury in India amount to one million casualties @onemilliondead. Half of trauma mortality in India occur in-hospital, and in-hospital trauma mortality in Indian hospitals was double that of high-income countries @indiantrauma. 
```

Comment:
Definitionerna här är en ormgrop och nu har jag fler frågor, som jag kanske inte behöver veta utan kan navigera runt. 

När jag ville presentera data för trauma i Indien omfattade den bästa statistiken olyckor (https://ncrb.gov.in/sites/default/files/ADSI-2019-FULL-REPORT.pdf) men rapporten har fler injury mechanisms än de i vår mode-of-injury kolumn. Trafik, fall och brännolyckor är ca 50% av dödsfallen i olyckor, kategorin "assault" är lite urspridd (firearms t.ex). Och då undrar jag om det skett någon selektion av project officers, om jag är off som förväntade mig att alla drunkningar förgiftningar osv skulle vara mer än de 5% som är NA/Other eller är drunkning&förgiftning inte definitionsmässigt trauma? mesh är öppet för tolkning.